editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -92,6 +94,10 @@ export const editorUIStore: StateCreator<
 
     toggleShowDataFields: () => set({ showDataFields: !get().showDataFields }),
 
+    showHelpText: false,
```

Comment:
We should also add this to the persist middleware like the other toggles so that a user's choice is saved.

To do this we just need to add it to the object passed to the middleware below this store.


```ts
{
    name: "editorUIStore",
    partialize: (state) => ({
      showSidebar: state.showSidebar,
      showTags: state.showTags,
      showImages: state.showImages,
      showDataFields: state.showDataFields,
    }),
  },
```

Reply:
Thanks for spotting - I'd forgotten this step but now updated !